server/organization/migrations/0001_initial.py

```diff
@@ -16,7 +17,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('org_id', models.AutoField(primary_key=True, serialize=False)),
                 ('org_name', models.CharField(max_length=256, unique=True)),
-                ('address', models.CharField(default='None', max_length=256)),
+                ('address', models.JSONField(null=True, blank=False)),
```

Comment:
What is the idea behind this field being nullable ?

Reply:
It should not be nullable, I forgot to set it to false. Every organization must at least have one location